src/vanilla/ClientModelExtensions.cs

```diff
@@ -682,6 +682,18 @@ public static void ConstructMapper(TSObject mapper, IModelType type, string seri
                 mapper.BooleanProperty("required", true);
             }
 
+            if (parameter?.IsXNullable != null)
+            {
+                if (parameter.IsXNullable.Value)
+                {
+                    mapper.BooleanProperty("nullable", true);
+                }
+                else
+                {
+                    mapper.BooleanProperty("nullable", false);
+                }
```

Comment:
This should probably be reduced to just the true scenario. If `nullable` is false, then we shouldn't include it in the mapper because not being included defaults to falsy, right?

Reply:
true, false and undefined are actually all treated differently for nullable. See https://github.com/RikkiGibson/ms-rest-js/blob/13da6db58bca7b109fad4888310a39a4c3a87798/lib/serializer.ts#L89

Reply:
Good now @daschult?